simplq/src/apis.js

```diff
@@ -1,21 +1,21 @@
-import axios from "axios"
+import axios from 'axios';
 
-const BASEURL = "http://localhost:3000/v1/"
+const BASEURL = 'http://localhost:3000/v1/';
 
 export const getQueuedItems = async (queueId) => {
-    const res = await axios.get(BASEURL + queueId)
-    if (res.data){
-        return res.data;
-    } else {
-        return [];
-    }
-}
+  const res = await axios.get(BASEURL + queueId);
```

Comment:
We were also sending an token here. I don't see it here.. strange. Are you making these changes on the latest master? Otherwise this is a huge change, and you might have merge issues later on. 🤔